src/components/Buttons/DonateButton.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Button } from '@mui/material';
+
+const DonateButton = ({ handleSubmit }) => {
+    return (
+        <Button
+            className="submit-btn"
```

Comment:
Can you change the class name to `donate-button`?

Reply:
That would make more sense. Fixed that in the component,and in CSS.